### PR TITLE
Modified Docs as per #10484

### PR DESCRIPTION
**Pull request by arafatkatze**
Adds doc as per #10484 
@asmeurer Please take a look.

### CODE REVIEW

File: doc/src/tutorial/gotchas.rst [hunk 1]
```diff
@@ -244,6 +244,10 @@ object.
     >>> type(1 + 1)
     <... 'int'>
 
+.. note::
+
+   On running the example above in sympy live,(1+1) is wrapped
+   by integer ,so it does not show the correct output.
```
Comment (asmeurer) [inline doc/src/tutorial/gotchas.rst]: `Integer`

Comment (asmeurer) [inline doc/src/tutorial/gotchas.rst]: Use the correct spacing around commas.

**Comment by Arafatk** [comment]
@asmeurer I have made the changes. Please take a look.

**Comment by Arafatk** [comment]
@asmeurer I have made the changes.

**Comment by aktech** [comment]
Looks good. Merging.

File: doc/src/tutorial/gotchas.rst [hunk 2]
```diff
@@ -244,6 +244,11 @@ object.
     >>> type(1 + 1)
     <... 'int'>
 
+.. note::
+
+   On running the example above in sympy live,(1+1) is wrapped
```
Comment (asmeurer) [inline doc/src/tutorial/gotchas.rst]: Space after comma

Comment (asmeurer) [inline doc/src/tutorial/gotchas.rst]: SymPy Live

Comment (aktech) [inline doc/src/tutorial/gotchas.rst]: Oops!

Comment (aktech) [inline doc/src/tutorial/gotchas.rst]: Will fix in a separate PR.

**Pull request merged**